lib/retriable_contexts/config.rb

```diff
@@ -0,0 +1,23 @@
+module Retriable
+  class Config
+    def self.validate_options(options)
```

Comment:
@kamui this method could just live in the main gem instead of in the bonus 'retriable_contexts`...  personally i tend to think it should, because i can't tell you how many times I or people I work with have done something like

```
Retriable.retriable(max_attempts: 10) do
  ...
end
```

using `max_attempts` (or `attempts`, or whatever) instead of `tries` accidentally, which the `Retriable` gem just sort of blithely ignores and ends up using defaults... which often doesn't get noticed for a long time because `Retriable` is mostly for retrying uncommon failures.

but i'll leave that decision up to you... i think we should either have strict validation of both configuration and calls to `Retriable.retriable` or no validation at all, so whatever you prefer I'll add to/remove from this PR before considering it final.

let me know what you prefer.